### DISCUSSION TITLE
Supervision support format

### DISCUSSION THREAD

**Discussion by ssvicnent** [roboflow/supervision#1759] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Does supervision only support three model format(inference\ultralytics\transformers)?

**Comment by onuralpszr** [comment]
Hello @ssvicnent 👋  No, we support more than 3 of them 

For Detections : https://supervision.roboflow.com/latest/detection/core/
For Keypoints: https://supervision.roboflow.com/latest/keypoint/core/

I hope that helps
